src/hooks/transactions/useSignTransactions.tsx

```diff
@@ -259,12 +261,26 @@ export const useSignTransactions = () => {
       return;
     }
 
-    const isLoggedInWithDifferentAccount = transactions.some((tx) => {
-      const sender = tx.getSender().toString();
-      return sender && address !== sender;
-    });
-
-    // Prevent signing transactions with different account
+    const senderAddresses = uniq(
```

Comment:
you could throw an error here if one of the senders is different. So if you consider only one sender, then you only fetch info about one account